lib/src/Uncertainty/Algorithm/Analytical/SystemFORM.cxx

```diff
@@ -223,6 +223,10 @@ void SystemFORM::run()
     eventProbability = -eventProbability + poincarreProbability;
   }
 
```

Comment:
Another option is to use a clever summation method as described [here](https://accurate-algorithms.readthedocs.io/en/latest/ch04summation.html) but it is another story... It could be implemented as a new method for Point, e.g. `Point::accurateSum()` or something like this.

Reply:
yes, lets drop that for now and remove the clip